### DISCUSSION TITLE
M1 Mac Mini

### DISCUSSION THREAD

**Discussion by ingebretsen** [Chia-Network/chia-blockchain#1430] [Q&A]
I'm having a hard time getting an M1 Mac Mini to complete a plot. Just wondering if anyone has had any luck yet? Wondering if I need to switch to an Intel setup...

**Comment by matt-k-wong** [comment]
It works fine on M1 Mac Mini.  What are your settings?

**Comment by CharlesWiltgen** [comment]
> I'm having a hard time getting an M1 Mac Mini to complete a plot.

The app has a cosmetic bug (at least for me) where the progress bar will be stuck at 100% until I restart, but plotting has actually finished. Is this what you're seeing?

If your progress bar has been 100% for a long time, and your temp drive is empty, and there are no temp files on the destination drive, then your plots are complete. I restart the app between groups of parallel plots, which then shows that the same plots are being farmed.
